anesthetic/plot.py

```diff
@@ -319,8 +366,8 @@ def hist_1d(ax, data, *args, **kwargs):
     return bars
 
 
-def contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
-    """Plot a 2d marginalised distribution as contours.
+def grid_kde_plot_2d(ax, data_x, data_y, *args, **kwargs):
+    """Plot a 2d marginalised distribution as contours on a grid.
```

Comment:
I don't like the name change here. When I'm looking for this function (scrolling or searching) I'm looking for "contour" as this is how I know it from matplotlib. I'd suggest something along the lines `contour_plot_2d_grid_kde` and `contour_plot_2d_sample_kde`.

Reply:
resolved in ec34419